### DISCUSSION TITLE
Comment box not showing up

### DISCUSSION THREAD

**Discussion by mbriggs** [pwntester/octo.nvim#242] [Q&A]
I am not sure if this is a bug or a misconfiguration. When i `Octo comment add` from a started review, nothing happens (no comment box, or error message). On nvim 0.6.1 and gh 2.4.0

**Comment by pwntester** [comment]
Hi, adding a new review comment on either side of the diff view should replace the other side with an octo buffer showing the diff hunk and an empty comment. No popups should be shown. This should only be the case if you place your comment on a line that belongs to a diff hunk. Can you maybe record a video of what happens and where you enter the add comment command?
